### PR TITLE
add wikicreator right

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -187,6 +187,10 @@
                 'noratelimit' => true,
                 'userrights' => true,
                 'userrights-interwiki' => true,
+             'wikicreator' => array(
```
Comment (Southparkfan) [inline LocalSettings.php]: The wikicreator array is not closed

File: LocalSettings.php [hunk 2]
```diff
@@ -187,6 +187,10 @@
...
+               	'createwiki' => true,
+               	'autopatroll' => true,
```
Comment (Southparkfan) [inline LocalSettings.php]: Should be autopatrol

Comment (JohnFLewis) [inline LocalSettings.php]: Should just be createwiki.

**Comment by Reception123** [comment]
Duplicated by #9

**Pull request closed without merging**